.travis.yml

```diff
@@ -1,42 +1,37 @@
 language: python
-
 addons:
   chrome: stable
-
 matrix:
   include:
-    - python: 3.6
-      env: TOXENV=py36
-    - python: 3.7
-      env: TOXENV=py37
-    - python: 3.8
-      env: TOXENV=py38
-    - python: 3.7
-      env: TOXENV=linting
-    - python: 3.7
-      env: TOXENV=formatting
-    - python: 3.7
-      env: TOXENV=docs
-
-
+  - python: 3.6
+    env: TOXENV=py36
+  - python: 3.7
+    env: TOXENV=py37
+  - python: 3.8
+    env: TOXENV=py38
+  - python: 3.7
+    env: TOXENV=linting
+  - python: 3.7
+    env: TOXENV=formatting
+  - python: 3.7
+    env: TOXENV=docs
 before_install:
-    - wget https://chromedriver.storage.googleapis.com/$(wget https://chromedriver.storage.googleapis.com/LATEST_RELEASE -q -O -)/chromedriver_linux64.zip
-    - unzip chromedriver_linux64.zip
-    - export PATH=$PATH:$PWD
-
+- wget https://chromedriver.storage.googleapis.com/$(wget https://chromedriver.storage.googleapis.com/LATEST_RELEASE
+  -q -O -)/chromedriver_linux64.zip
```

Comment:
isn't this supposed to be on the same line?

Reply:
Travis auto-formatted it for me, but can change it back if you want

Reply:
Travis auto-formatted it?

Reply:
I think I would prefer to change it back. Although most were indents, I also saw an added `matrix`.